src/query/v2/request_router.hpp

```diff
@@ -124,6 +127,10 @@ class RequestRouterInterface {
   virtual std::optional<storage::v3::LabelId> MaybeNameToLabel(const std::string &name) const = 0;
   virtual bool IsPrimaryLabel(storage::v3::LabelId label) const = 0;
   virtual bool IsPrimaryKey(storage::v3::LabelId primary_label, storage::v3::PropertyId property) const = 0;
+
+  virtual std::optional<std::pair<uint64_t, uint64_t>> AllocateInitialEdgeIds(io::Address coordinator_address) {
+    return {};
```

Comment:
Because of the mocked `RequestRouter` class, used in the tests.
This member function is not used there, but I can put the implementation there as well.

Reply:
plz do that